app/src/main/java/io/multy/ui/activities/MainActivity.java

```diff
@@ -126,6 +127,16 @@ private void setFragment(@IdRes int container, Fragment fragment) {
                 .commit();
     }
 
+    private void handleDonate() {
+        if (Prefs.getBoolean(Constants.PREF_APP_INITIALIZED)) {
+            ServerConfigResponse serverConfig = EventBus.getDefault().removeStickyEvent(ServerConfigResponse.class);
+            if (serverConfig != null) {
+                RealmManager.open(this);
+                RealmManager.getSettingsDao().saveDonation(serverConfig.getDonate());
```

Comment:
Would be more logically to put removing sticky event only after saving it to database. So it will guaranteed deleted only after saving to database.

Reply:
No.  In accordance with documentation by greenrobot.org/documentation/configuration/sticky-events/ this method of getting object is more preferable. We have two cases when database has been initialized then data have save on this method and when database is not initialized then we save donation info after initialize database by create new wallet.